dobi.yaml

```diff
@@ -420,11 +430,13 @@ job=run-node-identifier-integration-tests:
 
 # python jobs
 
+job=run-grapl-common-unit-tests:
+  use: grapl-common-build
+  command: /bin/bash -c "source venv/bin/activate && cd grapl_common/tests && ls && py.test -n auto -m 'not integration_test'"
+
 job=run-grapl-analyzerlib-unit-tests:
   use: grapl-analyzerlib-build
   command: /bin/bash -c "source venv/bin/activate && cd grapl_analyzerlib && py.test -n auto -m 'not integration_test'"
-  depends:
-    - grapl-analyzerlib-build
```

Comment:
I think this might be necessary given https://github.com/grapl-security/grapl/pull/226/checks?check_run_id=1023683435